restoration.go

```diff
@@ -0,0 +1,20 @@
+package flutter
+
+import (
+	"github.com/go-flutter-desktop/go-flutter/plugin"
+)
+
+type restorationPlugin struct{}
+
+// all hardcoded because theres not pluggable renderer system.
+var defaultRestorationPlugin = &restorationPlugin{}
+
+var _ Plugin = &restorationPlugin{} // compile-time type check
+
+// InitPlugin implements PluginGLFW
+func (p *restorationPlugin) InitPlugin(messenger plugin.BinaryMessenger) error {
+	channel := plugin.NewMethodChannel(messenger, "flutter/restoration", plugin.StandardMethodCodec{})
+	// Currently not supported. Used only to avoid warnings.
+	channel.HandleFunc("get", func(_ interface{}) (interface{}, error) { return nil, nil })
```

Comment:
I'm not sure about this, desktop CAN support `restoration` so it's better to either:
 - Implement it
 - Leave the warning

Reply:
Java implementation: https://github.com/huxiangjs/engine/blob/master/shell/platform/android/io/flutter/embedding/engine/systemchannels/RestorationChannel.java

Reply:
Humm, acutally, I don't know if this is a feature for desktop: https://api.flutter.dev/flutter/services/RestorationManager-class.html

> Mobile operating systems use the concept of state restoration to provide the illusion that apps continue to run in the background forever

What do you think?

Reply:
i think it's not intended for desktop